pkg/constants/constants.go

```diff
@@ -29,8 +29,13 @@ const (
 
 	RequestTimeout = 3 * time.Minute
 
-	FujiAPIEndpoint    = "https://api.avax-test.network"
-	MainnetAPIEndpoint = "https://api.avax.network"
+	DeployPublickyLocalMockEnvVar = "DEPLOY_PUBLICLY_LOCAL_MOCK"
+	FujiAPIEndpoint               = "https://api.avax-test.network"
+	MainnetAPIEndpoint            = "https://api.avax.network"
+
+	// this depends on bootstrap snapshot
+	LocalAPIEndpoint = "http://127.0.0.1:9650"
+	LocalNetworkID   = 1337
```

Comment:
There probably exists a constant for this (ANR?)

Reply:
No, this is network conf dependant